source/manual/alerts/low-available-disk-space.html.md

```diff
@@ -124,13 +126,11 @@ performing the maintenace tasks described below. Pass to 2ndline to deal
 with during working hours.
 
 Firstly see if you can remove any old logfiles, where dir is the
-directory where the large file lives: `find <dir> -mtime +45 -type f`
+directory where the large file lives: `find <dir> -mtime +45 -type f` (for
+files over 45 days old, for example).
 
 You can remove them by running: `find <dir> -mtime +45 -type f -delete`
 
-45 days is specified as this is how long we're obliged to keep logfiles,
```

Comment:
I agree with removing this 👍 looks like it was added 4 years ago in 0d2de769a9dd142a21a7558bbc1b2e5e2049ffe5, which has no additional context other than "they were imported from the opsmanual using a script", so the origins of this 'obligation' are unknown!

We should make sure this spreadsheet audit of logs continues to reflect reality, too: https://docs.google.com/spreadsheets/d/1tJ-m6da-nKnVhA1umeaIhFdSQij-bJE6EAGifyMqR7E/edit#gid=0